### ISSUE TITLE
[Bug]: Lots of false server offline messages on Coolify Cloud

### ISSUE THREAD

**Issue by drubb** [coollabsio/coolify#1542]
### Description

I'm using Coolify Cloud with one server. For two days now, I'm getting lots of messages of this kind:

> Coolify cannot connect to your server (...). Please check your server and make sure it is running.

> Your server (...) was offline for a while, but it is back online now. All automations & integrations are turned on again.

I don't see any problems related to my server. On the other side, this seems to happen since the last release (Beta 161), so there might be an issue with Coolify.

### Minimal Reproduction (if possible, example repository)

None, Coolify Cloud just sends lots of mails.

### Version

v4.0.0-beta.161

**Comment by andrasbacsai** [comment]
Thanks! I will check and finetune that feature. 

However, I've checked, and your server is the only one affected by this problem.

**Comment by drubb** [comment]
Thanks for taking a look. I've examined this further, turned out it's no issue with Coolify Cloud!

The root cause was an ongoing brute-force attack on my ssh port, causing lots of traffic. As a consequence, my provider repeatedly blocked all incoming traffic to this server for some time.
In addition, the fail2ban config on my server was a bit lax. I've hardened it now, and hopefully things should be good again.

I'll close this ticket, as Coolify can't do anything here. Thanks!
